### DISCUSSION TITLE
상태를 표현하는 변수 이름

### DISCUSSION THREAD

**Discussion by 0gunkim** [mi-gan-zi/FE-MiGanZi#87] [Code convention]
- 이미지 파일을 받아오고 상태를 업데이트하는 useState
```typescript
const [img, setImg] = useState<string>("");
```

- 이미지 유무 상태를 업데이트하는 usestate
```typescript
const [image, setImage] = useState(false);
```

초기값을 보면 유추할 수 있습니다 하지만 props로 사용된다면  `img`, `image` 둘다 이미지 파일이라고 생각됩니다.

- 상태를 알려주는 값(boolean 또는 특정 상태를 알려주는 문자) 변수 이름은 `isLoading`, `isImage`, `isImg`  is를 붙여서 사용한다 라고 배웠습니다.
```typescript
const [image, setImage] = useState(false);  =>  const [isImage, setIsImage] = useState(false);

```

상태를 표현(로딩, 프로미스, 값의 유무 상태 )하는 변수이름 다들 어떻게 사용하는지 궁금합니다

**Comment by pangkyu** [comment]
조금 구체적으로 적어서 프롭스로 넘길때 헷갈리지 않도록 사용하는 편입니다

boolean 값으로 상태를 체크하는 경우에는 **is**를 접두로 붙여서 사용하는 편이 좋다고 생각합니다

### https://velog.io/@cada/%EC%9E%90%EB%B0%94%EC%8A%A4%ED%81%AC%EB%A6%BD%ED%8A%B8-%EC%8A%A4%ED%83%80%EC%9D%BC-%EA%B0%80%EC%9D%B4%EB%93%9C-%EB%84%A4%EC%9D%B4%EB%B0%8D-%EC%BB%A8%EB%B2%A4%EC%85%98-%ED%8E%B8

**Reply by 0gunkim** [reply]
컨벤션 참고자료 고맙습니다
